website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -193,7 +193,10 @@ the [`sidecar_service`](/consul/docs/connect/proxies/deploy-sidecar-services) bl
 
 - `envoy_telemetry_collector_bind_socket_dir` - Specifies the directory where Envoy creates a Unix socket.
   Envoy sends metrics to the socket where a Consul telemetry collector can collect them.
-  The socket is not configured by default.
+  The socket is not configured by default. 
+  Enabling this feature sets Envoy's [`stats_flush_interval`](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-stats-flush-interval) to 1 minute by default to reduce the number of metric samples.
+  However, the`stats_flush_interval` default is not applied if a custom stats sink is configured using `envoy_statsd_url`,`envoy_dogstatsd_url` or `envoy_extra_stats_sinks_json`.
+  For a custom metric flush interval, configure a `envoy_stats_flush_interval` value explicitly.
```

Comment:
I'm suggesting the edit because I worry we'll call out a few stats sinks but a new setting will be added later, for a new sink, eg otel, and then the docs are a bit out of date because we haven't accounted for the new one.

I also removed mention of the sample count, which is sort of a result of an internal implementation details (that wouldn't apply for all TSDBs, for example)

```suggestion
  Enabling this sets Envoy's [`stats_flush_interval`](https://www.envoyproxy.io/docs/envoy/v1.17.2/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-stats-flush-interval) to one minute if `envoy_stats_flush_interval` is unset and if no other stats sinks are configured, like `envoy_dogstats_url`, for instance.
```

Reply:
100% thanks for the feedback, I think this is better worded 👍🏽 
Will let @hashicorp/consul-docs to weigh in as well